packages/angular/src/generators/ngrx/ngrx.spec.ts

```diff
@@ -208,4 +405,73 @@ describe('ngrx', () => {
 
     expect(devkit.formatFiles).not.toHaveBeenCalled();
   });
+
+  describe('unit tests', () => {
```

Comment:
Also a bit of a nitpick, sorry.
```suggestion
  describe('generated unit tests for user', () => {
```